app.js

```diff
@@ -100,3 +105,6 @@ oppiaGithubPages.controller('theExplorationGalleryController', function($scope)
 oppiaGithubPages.controller('learnerViewController', function($scope) {
 
 });
+oppiaGithubPages.controller('planningYourExplorationController', function($scope) {
```

Comment:
If in the future we might need interaction in pages they can help. Otherwise i can remove them if not needed at all.